tensorflow_datasets/testing/dummy_dataset/dummy_dataset_checksums.py

```diff
@@ -0,0 +1,4 @@
+from tensorflow_datasets.image_classification import cats_vs_dogs
+
+class DummyDatasetChecksums(cats_vs_dogs.CatsVsDogs):
```

Comment:
Rather than inheriting from an existing dataset, I think it's best to write a small dataset from scratch. 
Otherwise, this introduce a dependency `CatsVsDogs` -> `core tests` which seems to break tests boundary.
For instance, it's not be clear here if the tests would be passing because the original `CatsVsDogs` pass, or if tests are actually using the new relative path.